warp10/src/main/java/io/warp10/script/ext/capabilities/CAPGET.java

```diff
@@ -0,0 +1,50 @@
+//
+//   Copyright 2020  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.ext.capabilities;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
+
+public class CAPGET extends NamedWarpScriptFunction implements WarpScriptStackFunction {
+
+  public CAPGET(String name) {
+    super(name);
+  }
+
+  @Override
+  public Object apply(WarpScriptStack stack) throws WarpScriptException {
+    Object top = stack.pop();
+
+    if (top instanceof String) {
+      stack.push(Capabilities.get(stack, (String) top));
+    } else if (top instanceof List) {
```

Comment:
`top` could also be `null`, `Capabilities.get` would behave the same.

Reply:
Changed the behavior so only null returns the full set of capabilities, if a list is passed, only those in the list are.